### ISSUE TITLE
Review Architecture Wiki Page

### ISSUE THREAD

**Issue by hgputnam** [mountetna/monoetna#1160]
https://github.com/mountetna/monoetna/wiki/Data-Library-Back-End-Overview

This is mainly for Daryl but good for everyone to see. I am unsure what the role of the medusa host is. I am also not 100% sure that portainer lives on prometheus?

**Comment by coleshaw** [comment]
You're right, portainer now lives on dsco-mgr-1. I will give this a look and see if I have any additional details to add. Thanks!
